go.mod

```diff
@@ -14,8 +13,10 @@ require (
 	github.com/spf13/viper v1.12.0 // indirect
 	github.com/stretchr/testify v1.8.1
 	github.com/subosito/gotenv v1.4.0 // indirect
+	github.com/xmidt-org/sallust v0.2.1
 	github.com/xmidt-org/webpa-common v1.11.9
```

Comment:
Is this dependency still needed?

Reply:
oops, thanks! I forgot to run go mod tidy